cluster/gce/debian/master-helper.sh

```diff
@@ -40,8 +40,31 @@ function create-master-instance {
 
   write-master-env
   prepare-startup-script
-  gcloud compute instances create "${MASTER_NAME}" \
-    ${address_opt} \
+  create-master-instance-internal "${MASTER_NAME}" "${address_opt}" "${preemptible_master}"
+}
+
+function replicate-master-instance() {
+  local existing_master_zone="${1}"
+  local existing_master_name="${2}"
+  local existing_master_replicas="${3}"
+
+  local kube_env="$(get-metadata "${existing_master_zone}" "${existing_master_name}" kube-env)"
+  # Substitute KUBERNETES_MASTER_NAME to the replica's name.
+  kube_env="$(echo "${kube_env}" | grep -v "KUBERNETES_MASTER_NAME")"
+  kube_env="$(echo -e "${kube_env}\nKUBERNETES_MASTER_NAME: '${REPLICA_NAME}'")"
+  # Substitute INITIAL_ETCD_CLUSTER to enable etcd clustering.
+  kube_env="$(echo "${kube_env}" | grep -v "INITIAL_ETCD_CLUSTER")"
+  kube_env="$(echo -e "${kube_env}\nINITIAL_ETCD_CLUSTER: '${existing_master_replicas},${REPLICA_NAME}'")"
+  echo "${kube_env}" > ${KUBE_TEMP}/master-kube-env.yaml
+  get-metadata "${existing_master_zone}" "${existing_master_name}" cluster-name > ${KUBE_TEMP}/cluster-name.txt
+  get-metadata "${existing_master_zone}" "${existing_master_name}" startup-script > ${KUBE_TEMP}/configure-vm.sh
+
+  create-master-instance-internal "${REPLICA_NAME}"
+}
+
+function create-master-instance-internal() {
+  gcloud compute instances create "${1}" \
```

Comment:
Can you please create local variables for ${1} and ${2} (similarly to what you did in GCI) - this is much more readable then.